globals/matchmaking_utils.go

```diff
@@ -90,7 +91,8 @@ func RemoveClientFromAllSessions(client *nex.Client) {
 
 			rmcMessage := nex.NewRMCRequest()
 			rmcMessage.SetProtocolID(notifications.ProtocolID)
-			rmcMessage.SetCallID(0xffff0000)
+			rmcMessage.SetCallID(CurrentMatchmakingCallID)
+			CurrentMatchmakingCallID+=1
```

Comment:
Can just use the `Counter` type from `nex-go` here instead of managing this yourself